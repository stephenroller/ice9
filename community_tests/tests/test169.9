# trailing reference to proc parameter					fail

proc foo (a : int)

end

a 
:= 
0
;