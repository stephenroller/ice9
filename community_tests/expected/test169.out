line 7: undeclared variable: a